### DISCUSSION TITLE
Required Fields Module Missing

### DISCUSSION THREAD

**Discussion by jrock2004** [nvim-treesitter/nvim-treesitter#5918] [Q&A] [answered]
So my Lua diagnostics is complaining about missing modules but in readme I do not see that field. What am I missing? Should I just ignore this?

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/655716/7d7aa38d-a48b-4928-b4b3-4cceb785604e)

**Comment by clason** [comment, marked as answer]
You should just ignore this.
